nav2_smac_planner/src/collision_checker.cpp

```diff
@@ -86,6 +86,15 @@ bool GridCollisionChecker::inCollision(
   const float & angle_bin,
   const bool & traverse_unknown)
 {
+  // Check to make sure cell is inside the map
+  if (outsideRange(0, costmap_->getSizeInCellsX(), x) ||
```

Comment:
Since they're both `0` as min, you don't need to specify that as an input, just have `min` be `0` in the function itself.